modules/statics/src/base.ts

```diff
@@ -361,6 +361,17 @@ export enum UnderlyingAsset {
   GOLD = 'gold',
   GOT = 'got',
   GRT = 'grt',
+  GTAAVE18DP = 'GTAAVE18DP',
```

Comment:
i think to keep it consistent, we need to lowercase the quotes. e.g. `GTAAVE18DP = 'gtaave18dp'